contents/handbook/engineering/how-we-review.md

```diff
@@ -0,0 +1,42 @@
+---
+title: How we review PRs
+sidebar: Handbook
+showTitle: true
+---
+
+Almost all PRs made to repositories on PostHog will need a review from another engineer. We do this because almost every time we review a PR we find a bug, a performance issue, unnecessary code or UX that could have been confusing.
+
+## How to review
+
+1. Have a flick through the code.
+  - What to look for:
+    - Code that does the wrong thing or will produce bugs
```

Comment:
Perhaps worth adding specifically:

`- Are there any backwards-incompatible changes that might cause operational issues` - e.g. changing URLs of public endpoints that the frontend/users are using, doing irreversible/large db migrations together with large code changes instead of splitting it up etc.